packages/core/helper-plugin/src/utils/tests/stopPropagation.test.ts

```diff
@@ -29,8 +29,8 @@ describe('stopPropagation', () => {
 
   describe('StopPropagation component', () => {
     it('should render a div with stopPropagation properties', () => {
-      render(React.createElement(StopPropagation));
-      const divElement = screen.getByRole('button', { hidden: true });
+      const { getByRole } = render(React.createElement(StopPropagation));
+      const divElement = getByRole('button', { hidden: true });
```

Comment:
Just curious what's the difference here? I thought it was always preferable to use screen.

Reply:
I don't really have a strong preference, my intention here is to only import from `@tests/utils` so we don't import the wrong `render`. But we can re-export like I did with `act`. 

I think if you want to define the way to do it and implement it here as the basis im fine with it 😄 we just need to add to the contributor docs and probably an eslint rule 🤓